config.json

```diff
@@ -61,9 +61,7 @@
         "name": "Bird Watcher",
         "uuid": "60600885-8684-45c3-b8a9-a2f9e3ce2d97",
         "concepts": [
-          "arrays",
-          "for-loops",
-          "foreach-loops"
+          "arrays"
```

Comment:
One thing that worries me here is that the exemplar implementation of `bird-watcher` actually uses loops. And we can't make that a prerequisite here because that would lead to a dependency cycle, because the new exercise depends on `lists` which in turn depends on `arrays`

Reply:
I guess probably the solution would be to reformat `bird-watcher` as well to avoid needing loops? what do you suggest?

Reply:
> Maybe, I don't know to be honest. That would mean that the exercise is limited to just array creation and array indexing?

Yeah, could be something like this:
- How to create them
- How to access the elements in them
- How to access the length of the array
- How to create multi dimensional arrays
- Maybe how to make a copy of an array

Reply:
@kahgoh I would like to discuss this topic as well, before continuing with the changes to the loop concept, because we will probably have to reformat how the array concept is done as well. 

I think what I proposed seems okey, what do you think?

Reply:
Yeah, I think it be fine if we simplify the bird watcher exercise as you've suggested. If we want students the chance to  loop an over arrays, I think we could do it in Making the Grade if we wanted to.

Reply:
Okey I will prioritize that simplification then, and then we continue discussing the loop concept exercises, thanks for your opinions!